### ISSUE TITLE
1.8. Examine and fix accessibility

### ISSUE THREAD

**Issue by MaryBak** [crocs-muni/sec-certs#574]
**Description**
Currently, parts of the site are not up to WCAG AA standards, mostly due to insufficient contrast on elements, misstructured navigation, and other small issues. 

**Details for underlying issues and ideas for implementation:**
* WCAG AA standard should be used as a baseline.
* [Color Contrast Analyzer](https://chrome.google.com/webstore/detail/color-contrast-analyzer/dagdlcijhfbmgkjokkjicnnfimlebcll?hl=en) might be used.

**Additional context**
CCAT project
